components/ai_chat/core/common/features.h

```diff
@@ -38,15 +38,22 @@ extern const base::FeatureParam<double> kAITemperature;
 COMPONENT_EXPORT(AI_CHAT_COMMON)
 extern const base::FeatureParam<size_t> kMaxCountLargeToolUseEvents;
 
-// Whether should add indentation to page content structure for tool results.
-COMPONENT_EXPORT(AI_CHAT_COMMON)
-extern const base::FeatureParam<bool> kShouldIndentPageContentBlocks;
-
```

Comment:
I moved this lower so that the "large tool use events" related flags are grouped.